examples/esp32spi/minimqtt_pub_sub_blocking_topic_callbacks_esp32spi.py

```diff
@@ -13,14 +13,13 @@
 
 import adafruit_minimqtt.adafruit_minimqtt as MQTT
 
-### WiFi ###
+# Get WiFi details and broker keys, ensure these are setup in settings.toml
+ssid = getenv("CIRCUITPY_WIFI_SSID")
+password = getenv("CIRCUITPY_WIFI_PASSWORD")
+broker = getenv("broker")
+broker_port = getenv("broker_port")
 
```

Comment:
Maybe we provide defaults here too for port/broker, in addition to getting the aio_username (and I've grabbed key too)

```suggestion
ssid = getenv("CIRCUITPY_WIFI_SSID")
password = getenv("CIRCUITPY_WIFI_PASSWORD")
aio_username = getenv("ADAFRUIT_AIO_USERNAME")
aio_key = getenv("ADAFRUIT_AIO_KEY")
broker = getenv("broker", "io.adafruit.com")
broker_port = getenv("broker_port", 1883) # Port 1883 insecure, 8883 secure

```